packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php

```diff
@@ -34,7 +34,7 @@ public function  __construct(
         protected ExchangeRateRepository $exchangeRateRepository
     )
     {
-        $this->apiEndPoint = 'https://api.exchangeratesapi.io/latest';
+        $this->apiEndPoint = 'https://api.apilayer.com/exchangerates_data/convert';
```

Comment:
Add URL in services.php file.

~~~php
config('services.exchange_api.exchange_rates.url')
~~~

Reply:
And also make the `exchange-api` key to `exchange_api`.

Reply:
Done